inc/actions.php

```diff
@@ -150,21 +149,18 @@ function uploadcare_download(Uploadcare\File $file) {
 }
 
 /**
- * Add ajax upload handler
+ * Ajax upload handler - registers the new images in the Wordpress media library
  */
 add_action('wp_ajax_uploadcare_handle', 'uploadcare_handle');
 function uploadcare_handle() {
     // store file
     $api = uploadcare_api();
     $file_id = $_POST['file_id'];
     $file_url = $_POST['file_url'];
-    
-    $srcFile = $api->getFile($file_id);
-    $srcFile->updateInfo();
-    
-    $file = $api->uploader->fromUrl($file_url);
+
+    $file = new Uploadcare\File($file_url, $api);
+    $file->updateInfo();
```

Comment:
this request can be avoided.
we have to update `uploadcare-php` to update info on `store()` requests (API returns all the file data in response).

we can do it now or later (please add tickets to both repos then).